Task_DEV_3/Task_DEV_3/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace Task_DEV_3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter the length of the array :");
+            int arrayLength = Convert.ToInt32(Console.ReadLine());
+            if (arrayLength <= 5)
+            {
+                int[] inputNumbers = new int[arrayLength];
+
+                for (int i = 0; i < arrayLength; i++)
+                {
+                    Console.WriteLine($"Enter an array element № {i}");
+                    inputNumbers[i] = Convert.ToInt32(Console.ReadLine());
+                }
+
+                bool isPrimeNumber = true;
+
```

Comment:
It would be better to remove this empty line. It is not mistake but it can improve readability.